docs/src/prototypes/meetingOptions/styles.ts

```diff
@@ -0,0 +1,13 @@
+import { CSSProperties } from 'react'
+
+export const middleColumnStyles = {
+  width: '40%',
+  margin: '0 auto',
+}
+
+export const mainStyle: CSSProperties = {
+  backgroundColor: '#f3f2f1',
+  width: 'calc(100% - 250px)',
```

Comment:
this `250px` definitely should come as one of the value from theme's `siteVariables`